### PR TITLE
Goal votes

### PR DESCRIPTION
**Pull request by hector0513**

### CODE REVIEW

File: zomes/holo_acorn/code/src/lib.rs [hunk 1]
```diff
@@ -480,6 +480,18 @@ mod holo_acorn {
             address,
         })
     }
+    #[zome_fn("hc_public")]
+    fn update_goal(goal_vote:GoalVote , address: Address) -> ZomeApiResult<GetResponse<Goal>> {
```
Comment (Connoropolous) [inline zomes/holo_acorn/code/src/lib.rs]: Isn't there already an "update_goal" function? So maybe rename this one? That might be causing a bug

File: zomes/holo_acorn/code/src/lib.rs [hunk 2]
```diff
@@ -480,6 +480,18 @@ mod holo_acorn {
...
+        let app_entry = Entry::App("goal".into(), goal_vote.clone().into());
```
Comment (Connoropolous) [inline zomes/holo_acorn/code/src/lib.rs]: Change "goal" to "goal_vote"

**Comment by hector0513** [comment]
Yes

**Comment by hector0513** [comment]
https://github.com/h-be/acorn-hc/pull/53#issuecomment-560739085

**Pull request merged**